integration/tests/cook/test_basic.py

```diff
@@ -92,9 +92,12 @@ def test_max_runtime_exceeded(self):
         self.assertEqual('failed', job['instances'][0]['status'])
         self.assertEqual(2003, job['instances'][0]['reason_code'])
 
-    # load a job by UUID using GET /rawscheduler
+    def query_jobs(self, **kwargs):
```

Comment:
missing docstring, it's not clear to me how the function parameters are converted to url query parameters.